### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
In this part of the project, my partner Sam and I worked to simplify the main file in order to give it the single responsibility to start the code and show the initial menu.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -44,30 +43,37 @@ def create_person
...
+      puts 'Invalid Input'
+    end
+  end
+
+  def create_teacher(age, name)
+    puts 'Teacher\'s Specialization'
+    specialization = gets.chomp.to_s
+    @peoples.push(Teacher.new(specialization, age, true, name))
+    puts 'Teacher created successfully.'
+  end
```
Comment (HAMMAS-SALEEM) [inline app.rb]: - [ ] You can see that your methods are taking inputs using gets.chomp and also creating the objects means that your methods have two responsibilities. Kindly make your method single responsible to follow the requirements :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @juanmanuelbyc :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following GitFlow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -20,6 +20,13 @@ Technologies used : Git, Gitflows, Github, Rubocop
...
+👤 **Juan Diaz (Juan Diaz)**
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+- Twitter: [@twitter](https://twitter.com/juanmanueldiar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-díaz-5281b3111/)
+
+
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the issues page.
```
Comment (HAMMAS-SALEEM) [inline README.md:32]: - [ ] You have added a link to your license file at the end of your readme :blush: but you forgot to add the the license file :smile: but no problem you can add the file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @juanmanuelbyc and @samlorlah,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**